examples/with-markdoc/README.md

```diff
@@ -4,20 +4,27 @@ This example shows using [Markdoc](https://github.com/markdoc/markdoc) as top le
 
 ## Deploy your own
 
-Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example) or preview live with [StackBlitz](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/markdoc)
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example) or preview live with [StackBlitz](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/with-markdoc)
```

Comment:
@hideokamoto  I think I would prefer that these files are under `markdoc/`, not `with-markdoc/`

Reply:
In the Next.js examples dir, all integration examples names start from `with-XXX`.

- with-electron
- with-firebase
- with-stripe-typescript
- etc...

So I think `with-markdoc` is better.